Documentation/ContentObjects/Fluidtemplate/Index.rst

```diff
@@ -10,8 +10,12 @@ FLUIDTEMPLATE
 =============
 
 An object of type FLUIDTEMPLATE combines TypoScript with the Fluid
-templating engine. It has replaced the older and now deprecated
-:ref:`cobj-template` content object.
+templating engine. 
+
+.. versionchanged:: 11.0
+   The FLUIDTEMPLATE has replaced 
```

Comment:
there may be a "the" missing after the "replaced". Not quite sure.

Reply:
```suggestion
   The content object FLUIDTEMPLATE has replaced `TEMPLATE <https://docs.typo3.org/m/typo3/reference-typoscript/10.4/en-us/ContentObjects/Template/Index.html>`.
```

Reply:
so, hipe it can be used like that, then the following 2 lines have to be deleted.